README.md

```diff
@@ -1140,6 +1175,7 @@ copyright is owned by its contributors.
 [d-clap]: https://docs.rs/clap/2.24.2/clap/
 [d-error-chain]: https://docs.rs/error-chain/0.8.1/error_chain/
 [d-flate2]: https://docs.rs/flate2/0.2.19/flate2/
+[d-fnv]: https://doc.servo.org/fnv/
```

Comment:
Please make this link "https://docs.rs/fnv/1.0.3/fnv/". All docs on this page link to docs.rs